lib/core/src/Cardano/Wallet/Transaction/HttpBridge.hs

```diff
@@ -0,0 +1,237 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Cardano.Wallet.Transaction.HttpBridge
+    ( newTransactionLayer
+    ) where
+
+import Prelude
+
+import Cardano.Environment
+    ( Network (..), ProtocolMagic (..), network, protocolMagic )
+import Cardano.Wallet.Binary
+    ( toByteString )
+import Cardano.Wallet.CoinSelection
+    ( CoinSelection (..) )
+import Cardano.Wallet.Compatibility
+    ( HttpBridge )
+import Cardano.Wallet.Primitive.AddressDerivation
+    ( Depth (AddressK), Key, Passphrase (..), XPrv, XPub, getKey, publicKey )
+import Cardano.Wallet.Primitive.Types
+    ( Address (..)
+    , Coin (..)
+    , Hash (..)
+    , Tx (..)
+    , TxId (..)
+    , TxOut (..)
+    , TxWitness (..)
+    )
+import Cardano.Wallet.Transaction
+    ( ErrMkStdTx (..), TransactionLayer (..) )
+import Control.Monad
+    ( forM )
+import Data.ByteString
+    ( ByteString )
+import Data.Quantity
+    ( Quantity (..) )
+
+import qualified Cardano.Crypto.Wallet as CC
+import qualified Codec.CBOR.Encoding as CBOR
+import qualified Codec.CBOR.Write as CBOR
+import qualified Data.ByteString as BS
+import qualified Data.ByteString.Lazy as BL
+
+-- | Construct a 'TransactionLayer' compatible with Byron and the 'HttpBridge'
+newTransactionLayer :: TransactionLayer
```

Comment:
ok - I see this module is constructed by mostly moving code around

Reply:
Yes, it aggregates code from 'CoinSelection.Fee' and 'Primitive.Signing'